modules/client-configuration/pages/vhm-caasp.adoc

```diff
@@ -57,3 +58,441 @@ When using Salt or {productname} (either via UI or API) on any {caasp} nodes:
+    <media_url>http://$redhat_management_server/ks/dist/child/sle-module-containers15-sp1-pool-x86_64/$distrotree</media_url>
+    <name>sle-module-containers15-sp1-pool-x86_64 </name>
+   <product>sle-module-containers15-sp1-pool-x86_64</product>
+   <product_dir>/</product_dir>
+  </listentry>
+    <listentry>
+    <media_url>http://$redhat_management_server/ks/dist/child/sle-module-containers15-sp1-updates-x86_64/$distrotree</media_url>
+    <name>sle-module-containers15-sp1-updates-x86_64</name>
+   <product>sle-module-containers15-sp1-updates-x86_64</product>
+   <product_dir>/</product_dir>
+  </listentry>
+    <listentry>
+    <media_url>http://$redhat_management_server/ks/dist/child/suse-caasp-4.0-pool-x86_64-sp1/$distrotree</media_url>
+    <name>suse-caasp-4.0-pool-x86_64-sp1</name>
+   <product>suse-caasp-4.0-pool-x86_64-sp1</product>
+   <product_dir>/</product_dir>
+  </listentry>
+    <listentry>
+    <media_url>http://$redhat_management_server/ks/dist/child/suse-caasp-4.0-updates-x86_64-sp1/$distrotree</media_url>
+    <name>suse-caasp-4.0-updates-x86_64-sp1</name>
+   <product>suse-caasp-4.0-updates-x86_64-sp1</product>
+   <product_dir>/</product_dir>
+  </listentry>
+ </add_on_products>
+</add-on>
+ <!-- register -->
+  <suse_register>
+    <do_registration config:type="boolean">true</do_registration>
+    <install_updates config:type="boolean">true</install_updates>
+    <email><!-- replace this comment with an email address used for registration --></email>
+    <reg_code><!-- replace this comment with a CaaSP registration code --></reg_code>
+    <slp_discovery config:type="boolean">false</slp_discovery>
+    <addons config:type="list">
+      <addon>
+        <name>sle-module-containers</name>
+        <version>15.1</version>
+        <arch>x86_64</arch>
+      </addon>
+      <addon>
+        <name>caasp</name>
+        <version>4.0</version>
+        <arch>x86_64</arch>
+        <reg_code><!-- replace this comment with a CaaSP registration code --></reg_code>
+      </addon>
+    </addons>
+  </suse_register>
+
+  <scripts>
+  <chroot-scripts config:type="list">
+      <script>
+        <chrooted config:type="boolean">true</chrooted>
+        <filename>add_sles_sudo_rule.sh</filename>
+        <interpreter>shell</interpreter>
+        <source>
+<![CDATA[
+#!/bin/sh
+echo "Defaults:sles !targetpw
+sles ALL=(ALL,ALL) NOPASSWD: ALL" > /etc/sudoers.d/sles
+]]>
+          </source>
+      </script>
+    </chroot-scripts>
+<init-scripts config:type="list">
+      $SNIPPET('spacewalk/minion_script')
+    </init-scripts>
+</scripts>
+</profile>
+----
+
+== Manage a {caasp} Cluster With {productname}
+
+You can use {productname} to manage one or more existing {caasp} clusters.
+
+[NOTE]
+====
+Only {caasp}{nbsp}4 is currently supported.
+====
+
+
+Before you begin, ensure you have installed your {caasp} cluster.
+You will need to manually copy some configuration information from your cluster to the {productname} Server:
```

Comment:
The configuration should be copied to the elected Management Node.

Reply:
Fixed